runhouse/resources/hardware/cluster.py

```diff
@@ -1946,3 +1954,82 @@ def _folder_mv(
 
     def _folder_exists(self, path: Union[str, Path]):
         return self.client.folder_exists(path=path)
+
+    ###############################
+    # Cluster list
+    ###############################
+    @classmethod
+    def list(
+        cls,
+        show_all: Optional[bool] = False,
+        since: Optional[str] = None,
+        status: Optional[ClustersListStatus] = None,
+    ) -> Dict[str, List[Dict]]:
+        """
+        Returns user's runhouse clusters saved in Den and locally via Sky. If filters are provided, only clusters that are matching the
+        filters are returned. If no filters are provided, clusters that were active in the last 24 hours are returned.
+
+        Args:
+            show_all (bool, optional): Whether to list all clusters saved in Den. Maximum of 50 will be listed. (Default: False).
+            since (str, optional): Clusters that were active in the specified time period will be returned. Value can be in seconds, minutes, hours or days.
+            status (ResourceServerStatus, optional): Clusters with the provided status will be returned.
+
+        Examples:
+            >>> Cluster.list(since="75s")
+            >>> Cluster.list(since="3m")
+            >>> Cluster.list(since="2h")
+            >>> Cluster.list(since="7d")
+        """
+
+        try:
+            import sky
+
+            # get sky live clusters
+            sky_live_clusters = [
+                {
+                    "Name": sky_cluster.get("name"),
+                    "Cluster Type": "OnDemandCluster (Sky)",
+                    "Status": sky_cluster.get("status"),
```

Comment:
just confirming this is a string repr of the value, and not some internal sky object as the "status" value?

Reply:
yeah, good point. added a small change, it will look like this:
![image](https://github.com/user-attachments/assets/3e939e9d-6059-46c5-8df0-6c864fa1effe)